CSS/aboutStyle.css

```diff
@@ -287,4 +293,151 @@ ul {
     top: 15px;
     right: 35px;
   }
+
+  .desktop-banner {
+    display: none;
+  }
+
+  .desktop-headline {
+    display: none;
+  }
+
+  .logo-details-desktop {
+    display: none;
+  }
+
+  .desktop-footer {
+    display: none;
+  }
+}
+
+@media (min-width: 768px) {
+  .mobile-nav {
+    display: none;
+  }
+
+  .desktop-nav-banner {
+    display: flex;
+    background: #3e3e3e;
+    color: white;
+    flex-direction: row;
+    justify-content: end;
+    align-items: center;
+    height: 30px;
+    font-size: 12px;
+    gap: 10px;
+    padding-right: 4em;
+  }
+
+  .desktop-nav {
+    background: white;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    padding: 0 10em 0 10em;
+    gap: 10%;
+  }
```

Comment:
[REQUIRED] Kindly fix the horizontal scroll issue as suggested by previous reviewers. It usually happens when the width of an element on the page is more than the view width. ne easy way to detect which element it is is by giving all the elements on the page a border like this:
``` * {
border: solid 1px red;
}
``` 
Then moving the scroll bar to the right to see which item is popping out.
![image](https://github.com/demesameneshoa/First-Capstone-Project/assets/62938735/9f1316e2-8300-4df4-9d57-f40e371aa613)